sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala

```diff
@@ -123,6 +123,9 @@ class ParquetFileFormat
     // Sets compression scheme
     conf.set(ParquetOutputFormat.COMPRESSION, parquetOptions.compressionCodecClassName)
 
+    // Sets Parquet block size
+    conf.setInt(ParquetOutputFormat.BLOCK_SIZE, sparkSession.sessionState.conf.parquetBlockSize)
```

Comment:
For clarification, we are already able to set this via `parquet.block.size` but this PR proposes an alias for it, right?

Reply:
Yes, we are already able to set this via `parquet.block.size`, 
I think we should add this parameter into  "sql-programming-guide.md"

Reply:
I doubt if it is common enough to have an alias and document this in `sql-programming-guide.md`. Other configurations like `parquet.page.size`, `parquet.enable.dictionary` or `parquet.writer.version` are also rather similarly used as much as that configuration in my experience.

I would add this for now.

Reply:
Sounds reasonable.  I close it now， thanks